extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php

```diff
@@ -74,6 +74,8 @@ class WikiaRobots {
 	private $blockedPaths = [
 		'/d/u/', // User pages for discussions
 		'/fandom?p=', // Fandom old URLs
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)